Library/Formula/d-bus.rb

```diff
@@ -12,6 +12,8 @@ class DBus < Formula
     sha256 "08094b506bf81ce24aa9c38d9414b43726b35d19461440ecfc74d47d02298d50" => :mavericks
   end
 
+  depends_on "expat" unless OS.mac?
```

Comment:
`expat` isn't included in homebrew/dupes, but it's provided by OS X nonetheless, which is why this formula was missing the dependency.